internal/datacoord/services.go

```diff
@@ -952,6 +952,12 @@ func (s *Server) GetChannelRecoveryInfo(ctx context.Context, req *datapb.GetChan
 
 	channel := NewRWChannel(req.GetVchannel(), collectionID, nil, collection.Schema, 0) // TODO: remove RWChannel, just use vchannel + collectionID
 	channelInfo := s.handler.GetDataVChanPositions(channel, allPartitionID)
+	if channelInfo.SeekPosition == nil {
```

Comment:
Because we use describeCollectionInternal to `GetChannelRecoveryInfo`.
So a collection on creating status without SeekPosition can be seen at `GetChannelRecoveryInfo`, return error to avoid generate illeagal `RecoveryInfo` if SeekPosition is nil.